python/flink_agents/runtime/remote_execution_environment.py

```diff
@@ -133,10 +138,26 @@ class RemoteExecutionEnvironment(AgentsExecutionEnvironment):
     """Implementation of AgentsExecutionEnvironment for execution with DataStream."""
 
     __env: StreamExecutionEnvironment
+    __config: LocalConfiguration
 
     def __init__(self, env: StreamExecutionEnvironment) -> None:
         """Init method of RemoteExecutionEnvironment."""
         self.__env = env
+        self.__config = LocalConfiguration()
```

Comment:
Seems both RemoteExecutionEnvironment and LocalExecutionEnvironment loads `LocalConfiguration`, is it expected?  just from `LocalConfiguration` literately, it should be used for  LocalExecutionEnvironment only.